rapid_pro_tools/rapid_pro_client.py

```diff
@@ -726,6 +768,18 @@ def create_field(self, label, field_id=None):
 
             return rapid_pro_field
 
+    def create_group(self, name):
+        """
+        Creates a new contact group in a rapid_pro workspace.
+
+        :param name: group name.
+        :type name: str.
+        :return: the new group.
+        :rtype: temba_client.v2.types.Group
+        """
+
+        return self._retry_on_rate_exceed(lambda: self.rapid_pro.create_group(name=name))
```

Comment:
Does this fail if the group already exists?

Reply:
If the group exists, rapid pro will add a suffix 1,2.. and create a group with the modified name. Have updated the docstrings to highlight this thanks.